### PR TITLE
[QA] BOAC-2294: Removes feature flag restriction from topics API

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2294

### CODE REVIEW

File: src/components/sidebar/SearchForm.vue [hunk 1]
```diff
@@ -180,16 +180,21 @@ export default {
...
-    getTopics().then(data => {
-      this.each(data, topic => {
-        this.topicOptions.push({
-          text: topic,
-          value: topic
-        })
-      });
-    });
+    if (this.featureFlagEditNotes) {
+      this.getNoteTopics();
```
Comment (johncrossman) [inline src/components/sidebar/SearchForm.vue]: @lyttam - In `data()`, we init `topicOptions` with a non-empty array. Will the `b-form-select` render with one option?

**Comment by lyttam** [comment]
@johncrossman Actually, what we probably want is for the topics list to display in the search form even when note editing is turned off, right? So users can still search by topic.

**Comment by johncrossman** [comment]
@lyttam That's a good point. I would test locally by removing the `@feature_flag_edit_notes` on /api/notes/topics, set feature flag to false and verify search works fine. Thanks!

**Comment by johncrossman** [comment]
This looks good @lyttam  Ready for merge?

**Comment by lyttam** [comment]
@johncrossman yes, thanks

**Pull request merged**